### PR TITLE
add profilehooks

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

File: recipes/profilehooks/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
```
Comment (CJ-Wright) [inline recipes/profilehooks/meta.yaml]:
```suggestion
tests:
```

File: recipes/profilehooks/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  import:
```
Comment (CJ-Wright) [inline recipes/profilehooks/meta.yaml]:
```suggestion
  imports:
```

File: recipes/profilehooks/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+tests:
```
Comment (CJ-Wright) [inline recipes/profilehooks/meta.yaml]:
```suggestion
test:
```

**Comment by CJ-Wright** [comment]
@conda-forge/staged-recipes ready for review, anyone know what is going on with azure windows?

**Comment by chrisburr** [comment]
It's been broken for a while in staged-recipes (#7715). You can rely on Appveyor for testing Windows.

Looks good, LGTM 👍

**Comment by CJ-Wright** [comment]
thank you very much!

**Pull request merged**